common/fw-update-helper.cpp

```diff
@@ -209,12 +209,28 @@ namespace rs2
                     std::ofstream file(temp.c_str(), std::ios::binary);
                     file.write((const char*)flash.data(), flash.size());
                     log_backup_status = "Backup completed and saved as '"  + temp + "'";
-
                 }
             }
-            catch (...)
+            catch (const rs2::error& e)
```

Comment:
rs2::error inherits from std::exception... why have two separate handlers if you don't use the specific functions that rs2::error adds?

Reply:
Correct.
Dropping catch(rs2::error)